01-Primitives/05-Box-geometry.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>For cubic boxes ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>